featuretools/tests/primitive_tests/test_agg_feats.py

```diff
@@ -114,9 +108,14 @@ def count_generate_name(self, base_feature_names, relationship_path_name,
                                    where_str,
                                    use_prev_str)
 
-    Count = make_agg_primitive(count_func, [[Index], [Variable]], Numeric,
-                               name="count", stack_on_self=False,
-                               cls_attributes={"generate_name": count_generate_name})
+    Count = make_agg_primitive(
+        count_func,
+        [[ColumnSchema(semantic_tags={'foreign_key', 'category'})], [ColumnSchema()]],
```

Comment:
Same as prior comment regarding foreign key columns - I don't think `category` should be included...